example/issue_896/graphql.schema

```diff
@@ -0,0 +1,14 @@
+schema {
```

Comment:
Examples are there for people to understand how gqlgen works.

This should be in `codegen/testserver/issue896.graphql` and `extend schema {`. There should be plenty of examples in that directory.

Reply:
Thanks for the pointer, I have moved the schema file and added `extend` as you suggest.